All EIPIP Meetings/meeting 036.md

```diff
@@ -0,0 +1,346 @@
+ 
+**Lightclient**
+ I mean what's this why are things slowing down why can't we just figure out the formatting and then continue on.
+ 
+**Alita**
+ well it's because until the formatting is figured out we can't open any more PR's and so we can't get client people we can't get clients reviewing and stuff so in other words, my point is that the formatting is like a very high priority right now um I’m going to schedule a call with lightclient and Tomas for this weekend to figure this out if any if anybody else is interested in being on that please let me know and I will add you but yeah it's like it is essentially it's saying as we have like the raw information but we can't deliver and if you can't deliver then you just have a bunch of work in progress and then we just get to like I don't know I mean it's just it's not a very efficient like pipeline that we have going on right now.
+ 
+**MIcah**
+So I think it sounds like the solution is we need to resolve the formatting thing quickly is there anything we can do to expedite the solution?
+ 
+**Alita**
+ the best thing I can think of is just this weekend we schedule a call and get through it on that call like basically we just call until we figure it out so yeah it was supposed to be this past weekend but I had a crisis at work and that's resolved now so I will follow up with it but.
+ 
+**Lightclient**
+ I mean Tomasz not available asynchronously at all?
+ 
+**Alita**
+ I guess he is yeah I can also um message him. 
+ 
+**Lightclient**
+If you just ask him to chat in the jason rpc specs discord channel then.
+ 
+**Alita**
+Yeah, I think he's a little I think he tends to be a little bit unresponsive because he's very busy  I’m not totally sure uh but he tends to be a little bit slow at times and so I don't know I think a call would be more productive but what we could what I’ll do is I'll do an async as well as schedule a call just so that we hit both avenues and then we can just cancel the call if the async works.
+ 
+**Lightclient**
+ sounds good feel free to send me a link to the meeting and time.
+ 
+**Pooja** (cononical source for eip-43:57)
+cool so the next item is a canonical source for eips so I don't know uh brent if you may have any update.
+ 
+**Brent**
+Yeah that's a time issue for me I need to fork myself but I haven't got it's still on my list of things to do but haven't done anything recently so gotta push it again.
+ 
+**Pooja**
+ right uh like uh do you want me to keep it on recording meeting or should we take it off and one you have.
+ 
+**Brent**
+ yeah  let's try it at least one more meeting and i'll try and get something before the next meeting.
+ 
+**Pooja**(review action items-44:30)

```

Comment:
```suggestion
### Review action items
**Pooja** (44:30)

```

Reply:
Okay no worries I am in class but I will take care of it right after.